exchanges/binance/binance.go

```diff
@@ -23,8 +23,8 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
 )
 
-// Binance is the overarching type across the Binance package
-type Binance struct {
+// Exchange is the overarching type across the Exchange package
```

Comment:
suggestion: Across all exchanges `// Exchange encapsulates the core functionality for an exchange` or something similar